src/components/stack/index.test.jsx

```diff
@@ -0,0 +1,316 @@
+import React from "react";
```

Comment:
```suggestion
```

Can be deleted here. We're using the new jsx transform that doesn't require `import React` if you're not using it explicitly

Reply:
Done